api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md

```diff
@@ -99,7 +98,95 @@ The following code shows how you can customize the toolbar using this function.
     <dx-data-grid ...
         (onToolbarPreparing)="onToolbarPreparing($event)">
     </dx-data-grid>
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ...
+            @toolbar-preparing="onToolbarPreparing"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxDataGrid } from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid
+        },
+        methods: {
+            onToolbarPreparing(e) {
+                var toolbarItems = e.toolbarOptions.items;
+                // Modifies an existing item
+                $.each(toolbarItems, function(_, item) {
```

Comment:
Hmmm... jQuery.each in Vue?

Reply:
Tested our code converter. It's just a bug.